### DISCUSSION TITLE
Is it only me having Explorer Crashing if I right click Taskbar?

### DISCUSSION THREAD

**Discussion by ITDev93** [valinet/ExplorerPatcher#1939] [Q&A]
I have searched and searched and tried different solutions, uninstalling patcher, and re-installing it several times...

Explorer Patcher crashes for me and restarts if I try to right-click my taskbar.
![image](https://github.com/valinet/ExplorerPatcher/assets/22062468/0209d849-ff7d-4a38-954c-266936403d5b)

I have had this problem since the last Windows update (KB5029263 and KB5028948), which, according to my understanding, has issues...

Today, I saw an update for Explorer Patcher and I updated it, I thought the issue was resolved, but sadly, right-click crashes the patcher...

Now my Only Resort is to not display patcher "properties" in the taskbar...
![image](https://github.com/valinet/ExplorerPatcher/assets/22062468/7145d762-5abc-4615-9a43-40975b0364d4)

I do not have preview updates, I am sticking to Normal updates and these are the last updates I got which caused Explorer Patcher to crash...
![image](https://github.com/valinet/ExplorerPatcher/assets/22062468/9ac60dcf-6e7d-4a76-98ce-2ef62473bafc)

My Taskbar settings are default:
![image](https://github.com/valinet/ExplorerPatcher/assets/22062468/ce71c974-3327-4cbd-8513-8b1121464933)

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1828

Please close this one.

Possible fix: In ExplorerPatcher Properties, go to Advanced and tick "Do not change the taskbar context menu"

**Reply by ITDev93** [reply]
How is that possible?

I have gone through the entire thing and don't see any similar cases as mine, Windows 11 Taskbar is broken with Explorer Patcher...

I applied this (https://github.com/valinet/ExplorerPatcher/issues/1828#issuecomment-1675937607):
> > @Vipin2003 : #1834 mentions a few bugs but no systematic crash like this issue.
> 
> You tried to access explorer or R-clicked on taskbar after immediate update.. And you haven't wait for symbols to download. That's why, this happens.
> 
> To fix this issue, follow these steps now:
> 
> 1. Uninstall EP completely and make a restart.
> 2. After restart, delete all EP reg. keys : `HKEY_CURRENT_USER\Software\ExplorerPatcher` and `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher`
> 3. make another restart.
> 4. Install EP again
> 5. wait for Symbols to be downloaded. Don't even touch taskbar. It will take around 5-6 min
> 6. After symbols notification, make one last restart.
> 
> After this, everything will work fine.

After deleting everything, I did not import my settings, I used it as it is. It had Windows 10 taskbar right-click did not crash, changed it to Windows 11, restarted Explorer patcher, right-click the taskbar, crash...

So I am importing my settings and removing Explorer Patcher properties from the taskbar right-click till hopefully an update fixes this...

**Comment by Rafael-Menis** [comment]
> How is that possible?
> 
> I have gone through the entire thing and don't see any similar cases as mine, Windows 11 Taskbar is broken with Explorer Patcher...
> 
> I applied this ([#1828 (comment)](https://github.com/valinet/ExplorerPatcher/issues/1828#issuecomment-1675937607)):
> 
> > > @Vipin2003 : #1834 mentions a few bugs but no systematic crash like this issue.
> > 
> > 
> > You tried to access explorer or R-clicked on taskbar after immediate update.. And you haven't wait for symbols to download. That's why, this happens.
> > To fix this issue, follow these steps now:
> > 
> > 1. Uninstall EP completely and make a restart.
> > 2. After restart, delete all EP reg. keys : `HKEY_CURRENT_USER\Software\ExplorerPatcher` and `HKEY_CURRENT_USER\Software\Microsoft\Windows\CurrentVersion\Explorer\ExplorerPatcher`
> > 3. make another restart.
> > 4. Install EP again
> > 5. wait for Symbols to be downloaded. Don't even touch taskbar. It will take around 5-6 min
> > 6. After symbols notification, make one last restart.
> > 
> > After this, everything will work fine.
> 
> After deleting everything, I did not import my settings, I used it as it is. It had Windows 10 taskbar right-click did not crash, changed it to Windows 11, restarted Explorer patcher, right-click the taskbar, crash...
> 
> So I am importing my settings and removing Explorer Patcher properties from the taskbar right-click till hopefully an update fixes this...

I had a similar issue as yours, and following those directions fixed it. I'm able to use the windows 11 taskbar without crashes.
